pom.xml

```diff
@@ -153,6 +153,22 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jdeps-plugin</artifactId>
+                <version>3.1.2</version>
+                <executions>
+                    <execution>
+                        <goals>
```

Comment:
The jdeps shouldn't be added into the production pom. It's only for validations

Reply:
Ok, thanks, that was confusing since I saw other repos with the plugin. i'll add a task to remove it from the others too then.